tests/auto/perfdata/vector_static_gcc/perf.data.expected.txt

```diff
@@ -381,6 +381,7 @@ vector_static_g	349054	349054	65149.472738182
 vector_static_g	349054	349054	65149.472968985
 	cycles: 296517
 
+	41b3b0
```

Comment:
the changes to the testdata  here are only related to the first patch in the series (zero size symbols) - merge them into that one

Reply:
Fixed